src/bosh_aws_cpi/lib/cloud/aws/aws_provider.rb

```diff
@@ -19,16 +19,15 @@ def initialize(aws_config, logger)
       @ec2_resource = Aws::EC2::Resource.new(client: @ec2_client)
     end
 
-    def aws_accessible?
-      # make an arbitrary HTTP request to ensure we can connect and creds are valid
-      @ec2_resource.subnets.first
-      true
-    rescue Seahorse::Client::NetworkingError => e
-      @logger.error("Failed to connect to AWS: #{e.inspect}\n#{e.backtrace.join("\n")}")
-      err = "Unable to create a connection to AWS. Please check your provided settings: Region '#{@aws_config.region || 'Not provided'}', Endpoint '#{@aws_config.ec2_endpoint || 'Not provided'}'."
-      cloud_error("#{err}\nIaaS Error: #{e.inspect}")
-    rescue Net::OpenTimeout
-      false
+    def self.with_aws
+      begin
+        yield
+      rescue Seahorse::Client::NetworkingError => e
+        err = "Unable to create a connection to AWS. Please check your provided settings: Region and Endpoint."
```

Comment:
We are now not logging an error anymore with @logger and changed the cloud_error message slightly.
This happened because `with_aws` is a static method. Do you think this is a problem?